### PR TITLE
Adds and re-arranges NT clothes and product disk designs into two separate disks

### PR DESCRIPTION
**Pull request by GoodG1448**
## About The Pull Request

NT now has two disks that should be located in their reactor room:
-NeoTheology Bio-Fabric Designs: _**NT voidsuits/armors, coats, clothes, belts, holsters, NT encrypt key and misc clothing accessories**_
-NeoTheology Bioprinter Products and Utilities: _**Foodstuffs, fertilizer bottles, medical stuff and cleaning utilities**_ (Adds NT brand cleaner and weedkiller grenades that can be printed from the bioprinter using biomatter, water and other materials)

1a 
2a

The material cost for the NT voidsuits are open to change if they seem cheap.
Also, made weedkiller grenades less shit at their job.

**_(Note: I am leaving the disk mapping placement to someone else that has properly set up their mapmerger and their gitdesktop, Win10 is a bitch for me. The clothes disk should go right on top of the table next to the bioprinter at the bioreactor room)_**

## Why It's Good For The Game

NT should be able to properly outfit people that desires to be converted or latejoining NT church members if the equipment is missing/lacking for whatever reason. Makes the NT product disk less cluttered and the new grenades allows custodians to have more tools to deal with mushrooms and filth.

## Changelog
:cl:
tweak: NT clothes and product designs are now split between two different disks
add: NT can now print armor, coats, jackets and clothes of their own
add: NT brand cleaner and weedkiller grenades are now available from NT product disk
/:cl:

### CODE REVIEW

**Comment by ArmolitskiyCarpoeb** [comment]
nice!

**Comment by fernandos13** [comment]
YOU did it NICE

File: code/datums/autolathe/biomatter.dm [hunk 1]
```diff
@@ -31,31 +31,57 @@
 	materials = list("biomatter" = 7.5)
 //[/NUTRIMENTS]
 
-//[THINGS]
-/datum/design/bioprinter/wallet
-	name = "Wallet"
-	build_path = /obj/item/weapon/storage/wallet
+//[CLOTHES, ARMOR AND ACCESORIES]
+datum/design/bioprinter/nt_clothes/acolyte_armor
```
Comment (mattroks101) [inline code/datums/autolathe/biomatter.dm:35]:
```suggestion
/datum/design/bioprinter/nt_clothes/acolyte_armor
```
You dropped this, king.

**Review by mattroks101 — DISMISSED** [review]

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 2]
```diff
@@ -292,17 +292,23 @@
...
 	beakers += B1
 	beakers += B2
 	icon_state = "grenade"
 
+/obj/item/weapon/grenade/chem_grenade/antiweed/nt_antiweed
+	name = "NeoTheology \"Kudzu Killer\""
+	desc = "NT brand weedkiller grenades. Designed to deal with Kudzu infestations back in New Rome. Mixes toxic biomatter with plasticides for great results"
+	icon_state = "foam"
+	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTIC = 2, MATERIAL_BIOMATTER = 5)
+	matter_reagents = list("water" = 30)
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm:310]: These newly added grenades only have water

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 3]
```diff
@@ -325,6 +331,13 @@
 	beakers += B1
 	beakers += B2
 
+/obj/item/weapon/grenade/chem_grenade/cleaner/nt_cleaner
+	name = "NeoTheology \"Cleanse Capsule\""
+	desc = "NT brand cleaner grenades. Designed to deal with Biogenerator accidents and the aftermaths of gang wars inside the New Rome slums."
+	icon_state = "foam"
+	matter = list(MATERIAL_STEEL = 3, MATERIAL_BIOMATTER = 5)
+	matter_reagents = list("water" = 30)
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm:339]: Ditto here, water chem.

**Comment by GoodG1448** [comment]
Uh, that's the point, I even tested them and they work as intended.

When you use the NT Products disk you can print those grenades by using plastic, steel, biomatter and water (30 units to be specific, that has to be inside the bioprinter's beaker). Once printed, you get the grenades with no extra water, they are basically just the same as the Aester's version of them.

Unless you mean that they don't require the other chemicals to make the reaction, if that's the case then let me explain: Lorewise,  It basically uses the same logic behind the NT cleaner carbine cleaning modes. Biomatter in this case is added to the print requirements to act as a cleaning agent or a weedkiller agent.

And god no, why would I add surfactant as a requirement to print them. That shit is nigh impossible to get while playing as a NT church member unless you decided to raid Mobious for the chem dispencers.

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**